yarpl/include/yarpl/flowable/Subscribers.h

```diff
@@ -76,14 +76,28 @@ class Subscribers {
     }
 
     void onNext(T value) override {
-      next_(std::move(value));
+      if (userError_) {
+        return;
+      }
+      try {
+        next_(std::move(value));
+      } catch (const std::exception& exn) {
+        userError_ = true;
+        onError(folly::exception_wrapper{std::current_exception(), exn});
```

Comment:
It seems like you'd want to `cancel()` the subscription here, otherwise the publisher is going to keep sending you `onNext()` even though the subscriber has already been terminated.